src/app/components/LiveLabel/index.tsx

```diff
@@ -50,29 +50,47 @@ const LiveLabel = ({
   }
 
   const circle = [
-    [styles.liveLabelCircle, className === 'first-promo' && styles.firstPromo],
+    styles.pulseOuterCircle,
+    className === 'first-promo' && styles.firstPromo,
   ];
 
   return (
-    // The id below is a temporary fix for the a11y nested span's bug experienced in TalkBack, refer to the following issue: https://github.com/bbc/simorgh/issues/9652
-    // eslint-disable-next-line jsx-a11y/aria-role
-    <span id={id} role="text">
+    <React.Fragment>
```

Comment:
I can't remember if I asked this before, but does this work?

You'll need to include `/* @jsxFrag React.Fragment */` at the top of the file

```suggestion
    <>
```

Reply:
Yes, thanks! I used the empty fragment first and it WHINGED but I'd forgotten about adding the comment at the top of the file, doh 😄